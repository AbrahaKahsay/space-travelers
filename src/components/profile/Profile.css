.container {
    width: 80%;
    justify-content: center;
    margin: 0.5rem 6rem;
    border-spacing:30px 0px;
  }
  
  .container td,
  .container th {
    width: 50%;
    text-align: left;
  }
  
  .container tr:nth-child(even) {
    background-color: #f2f2f2;
  }
  
  .container th {
    padding-top: 5px;
    padding-bottom: 5px;
  }
  
  .rockets,
  .missions {
      padding: 0.25rem 2rem;
  }