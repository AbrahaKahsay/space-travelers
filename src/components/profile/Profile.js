import React from "react";

const Profile = ()=>{
    return(
        <div><p>This profile section</p></div>
    )
}

export default Profile;