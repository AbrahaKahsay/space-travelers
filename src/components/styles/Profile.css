.profiles {
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
}
