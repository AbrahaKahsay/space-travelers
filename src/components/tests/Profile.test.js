import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import Profile from '../../components/Profile/Profile'
import store from '../../Redux/configureStore';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const { container } = render(
      <MemoryRouter>
        <Provider store={store}>
          <Profile />
        </Provider>
      </MemoryRouter>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});