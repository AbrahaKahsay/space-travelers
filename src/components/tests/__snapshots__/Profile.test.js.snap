// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<table
  class="cont"
>
  <tbody>
    <tr>
      <th>
        <h3>
          My Missions
        </h3>
      </th>
      <th>
        <h3>
          My Rockets
        </h3>
      </th>
    </tr>
    <tr>
      <td />
      <td />
    </tr>
  </tbody>
</table>
`;
